docs/source/bricks.rst

```diff
@@ -901,8 +901,11 @@ Example:
 --------------------------
 
 Convert a list of ``Text`` elements to an `Argilla Dataset <https://docs.argilla.io/en/latest/reference/python/python_client.html#python-ref-datasets>`_.
-The type of Argilla dataset to be generated can be specified with `argilla_task` parameter. Currently, only ``text_classification``
-task type is supported.
+The type of Argilla dataset to be generated can be specified with ``argilla_task``
+parameter. Valid values for ``argilla_task`` are ``"text_classification"``,
+``"token_classification"``, and ``"text2text"``. If ``"token_classification"`` is selected
+and ``tokens`` is not included in the optional recrod kwargs, the ``nltk`` word tokenizer
```

Comment:
```suggestion
and ``tokens`` is not included in the optional kwargs, the ``nltk`` word tokenizer
```
I think this is a typo of record, but I'm not sure what's meant by "optional record kwargs"? I'm suggesting just removing it but that may not capture the intent here.

Reply:
Thanks, yeah I think it looks fine with 💿 removed.